jobs/JGDAS_ENKF_FCST

```diff
@@ -15,7 +15,7 @@ export CDUMP=${RUN/enkf}
 
 export CASE=${CASE_ENS}
 
-YMD=${PDY} HH=${cyc} generate_com -rx COM_TOP
+YMD=${PDY} HH=${cyc} generate_com -rx COM_TOP COM_CONF
```

Comment:
I think this should be in `exgdas_enkf_fcst.sh`.
For now this is OK to get the test passing, but leave a `TODO` to move this here.

Reply:
I am not sure I follow here, but all of the `com` directory structure definitions are in `generate_com` and passing `MEMDIR` **should** define the respective ensemble member path. 

I apologize if I am misunderstanding what is being discussed here or this is known.

Reply:
@aerorahul OK. This is what I was talking about, but was looking in the incorrect space.

Once we merge this, I can open a new issue and update this.

Reply:
@HenryWinterbottom-NOAA I think this is a small enough change that you can make in this PR itself.
Essentially delete this line from `JGDAS_ENKF_FCST` and add `COM_CONF` on line 134 of `exgdas_enkf_fcst.sh`.

Reply:
Let's just do it now so it is correct before merging.

Reply:
@aerorahul @WalterKolczynski-NOAA Done.